.deploy.bash

```diff
@@ -6,7 +6,7 @@ set -u
 # PYPI_USERNAME - (Required) Username for the publisher's account on PyPI
 # PYPI_PASSWORD - (Required, Secret) Password for the publisher's account on PyPI
 
-cat << EOF >> ~/.pypirc
+cat << EOF > ~/.pypirc
```

Comment:
Why is this necessary? Are you worries that pypirc might already exist in some cases?

Reply:
Ah I see. I just saw the commit message. All good, ignore this question.